apps/music/js/music.js

```diff
@@ -160,88 +160,116 @@ var App = (function() {
     }
   }
 
-  function showCurrentView(callback) {
-    // We need AlbumArtCache.getCoverURL() to display thumbnails; it might not
-    // have been loaded yet, so make sure we load it first. This should prevent
-    // us from having to worry about loading it anywhere else in the code, since
-    // showCurrentView is called pretty early in the startup process.
```

Comment:
It looks like you've changed your mind about that and are now using LazyLoader before every use...